database/migrations/012_friend_requests_table.up.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE `friend_requests` (
```

Comment:
Note: Bump the migration number due to the recent pr that was merged before this.

Reply:
resolved in 42b41d8